detox/test/e2e/detox.config.js

```diff
@@ -108,7 +109,7 @@ const config = {
       headless: Boolean(process.env.CI),
       gpuMode: process.env.CI ? 'off' : undefined,
       device: {
-        avdName: 'Pixel_3a_API_34'
+        avdName: process.env.CI ? 'Pixel_3a_API_34' : 'Pixel_3a_API_35',
```

Comment:
Nahhh this doesn't make sense. Let's just create this emulator in advance and start using it locally.